demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1879,3 +1893,78 @@ def execute_tests(self):
             raise
         finally:
             self.build_context.logging_module.execute_logs()
+
+
+def replace_external_playbook_configuration(client: DefaultApi, external_playbook_configuration: dict,
+                                            logger_module: logging.Logger = logging.getLogger('demisto-sdk')):
+    """ takes external configuration of shape {"playbookID": "Isolate Endpoint - Generic V2",
+                                               "input_parameters":{"Endpoint_hostname": ["simple", "test"]}
+        and changes the specified playbook configuration to the mentioned one.
+        Returns (Whether the Playbook changed, The values to restore, the path to use when restoring)
```

Comment:
Please add in the docstring the case(s) where a restore will be needed.

Reply:
Restore is needed in case the inputs was changed, Do you think I should be more specific on what might fail the change?

Reply:
I see it's just for that, but it'll be easier for a dev who reads this code to understand when restore is needed.